fearless/Modules/SoraCard/KYCMain/KYCMainPresenter.swift

```diff
@@ -34,20 +34,23 @@ extension KYCMainPresenter: KYCMainViewOutput {
     }
 
     func didTapGetMoreXor() {
-        let languages = localizationManager?.selectedLocale.rLanguages
+        guard let chainAsset = interactor.xorChainAssets.first(where: { chainAsset in
```

Comment:
maybe it's make sense to store main chain xor chainasset in interactor ? you can set it once you received chainAssets